source/_includes/installation/operating_system.md

```diff
@@ -48,6 +48,33 @@ We will need a few things to get started with installing Home Assistant. Links b
 
 {% endif %}
 
+{% if page.installation_type == 'generic-x86-64' %}
+
+<div class='note'>
+<b>Prerequisites</b>
+
+This guide assumes that you have a dedicated generic x86 PC (typically an Intel or AMD based system) available to run exclusively Home Assistant OS. The system must be 64-bit capable and able to boot using UEFI. Pretty much all systems produced in the last 10 years support the UEFI boot mode.
+</div>
+
+### Configure the BIOS
+
+To boot Home Assistant OS the BIOS needs to have UEFI boot mode enabled and Secure Boot disabled. This following screenshots are from a 7th generation Intel NUC system. The BIOS menu will likely look different on your systems. However, the options should still be present and named similarly.
+
+1. Enter the BIOS using the can enter using the `F2` key (on some systems this might be `Del`, `F1` or `F10`).
+![Enter BIOS using F2, Del, F1 or F10 key](/images/installation/intel-nuc-enter-bios.jpg)
+
+1. Make sure the UEFI Boot mode is enabled
+![Enable UEFI Boot mode](/images/installation/intel-nuc-uefi-boot.jpg)
+
+1. Disable Secure Boot
+![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)
+
+As a next step we need to write the Operating System image to the installation media. Depending on your system this can be a S-ATA hard disk, S-ATA SSD, a M.2 SSD or even a eMMC. Home Assistant OS has no integrated installer. You need to use your Desktop computer (e.g. by using a USB to S-ATA adapter) or boot a live operating system on your target system to install Home Assistant OS.
```

Comment:
```suggestion
As a next step, we need to write the Operating System image to the installation media. Depending on your system this can be a S-ATA hard disk, S-ATA SSD, a M.2 SSD or even a eMMC. Home Assistant Operating System has no integrated installer. You need to use your Desktop computer (e.g. by using a USB to S-ATA adapter) or boot a live operating system on your target system to install Home Assistant Operating System.
```